src/components/YearPicker.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+<div class="container outer">
+  <div class="container box">
+    <p class="year">{{ this.year }}</p>
+    <button class="btn btn-primary float-left" @click="onClick($event, mode=true)">
+      &lt; {{ this.prevYear }}
+    </button>
+    <button class="btn btn-primary float-right" @click="onClick($event, mode=false)">
+      {{ this.nextYear }} &gt;
+    </button>
+  </div>
+</div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component
+export default class YearPicker extends Vue {
+  public year: number = new Date().getFullYear();
+
+  public prevYear: number = this.year - 1;
+
+  public nextYear: number = this.year + 1;
+
+  onClick(event: Event | undefined, mode: boolean): void {
+    // if true; previous year button is clicked
+    if (mode === true) {
```

Comment:
can we change this onClick handler to receive a number instead of a boolean argument?

that way, we don’t need an if/else - we could just add the number coming in.

to enforce the limits, we could disable the prevYear/nextYear buttons when current year is min or max

does that make sense?